app/dashboard/templatetags/lineup.py

```diff
@@ -0,0 +1,6 @@
+from django import template
+
+register = template.Library()
+
+@register.filter
+def lineup(ls): return ', '.join(ls[:-1])+' and '+ls[-1] if len(ls)>1 else ls[0]
```

Comment:
Please make this and the other files pep8 compliant.  You can run yapf against your files or inspect with a linter.

Reply:
@mbeacom I inspected with a linter, just seeing long line comments now. Please let me know if you see anything else in notifications.py